app/car.py

```diff
@@ -0,0 +1,6 @@
+class Car:
+    def __init__(self, car: dict) -> None:
+        self.fuel_consumption = car["fuel_consumption"]

```

Comment:
Yeah, why even bother with those "brand" things? Not like they change anything. :)
It's just a comment. The only reason for reject is the other one.